test/server/lib/DocApi.ts

```diff
@@ -2988,21 +2988,22 @@ function testDocApi(settings: {
         assert.deepEqual(tarUploadResp.data, { error: "File is not a valid .tar" });
       });
 
-      it("POST /docs/{did}/copy fails when the document has external attachments", async function () {
+      it("POST /docs/{did}/copy doesn't throw when the document has external attachments", async function () {
         const worker1 = await userApi.getWorkerAPI(docId);
-        await assert.isRejected(worker1.copyDoc(docId, undefined, 'copy'), /status 400/);
+        await worker1.copyDoc(docId, undefined, 'copy');
```

Comment:
Shouldn't we explicitly do `assert.doesNotThrow` here?

Reply:
Don't think we need to? If it throws, the exception causes the test to fail. I think `assert.doesNotThrow` is primarily for checking that the thrown error isn't a specific error.